### ISSUE TITLE
x86 version doesn't look into the redist32 sub-dir for the Oracle files

### ISSUE THREAD

**Issue by avogelba** [avogelba/uLister#1]
X64 Version works fine,
but it looks like the x86 version doesn't look into the redist32 sub-dir for the Oracle files.

**Comment by NSP-0123456** [comment]
in util.cpp @ getlibpath function you have the switch.
It is using negation of **_M_X64** and **__amd64__**  macro to detect compiled version. You can probably use another macro to force 32Bit only when compiling 32bit.
Another solution is to use _WIN32 or _WIN64 macros

**Comment by yozhic** [comment]
Yes, unfortunately the x86 version does not work at all. 32-bit Total Commander is still relevant as it works more stable in some cases and there are some plugins for which 64-bit analogs have not been made yet. Please fix the x86 version.

**Comment by avogelba** [comment]
The problem is somehere else it seems. ListLoad is never called. but why?
I can only take a working project and compile it 1:1 and see its its a compiler problem
